src/shared/utils/get-key.js

```diff
@@ -0,0 +1,12 @@
+const buildFragmentMatcher = lowerLabel => definition => lowerLabel.includes(Object.values(definition)[0])
```

Comment:
Please refactor into a proper function expression with closures if necessary; this kind of code is high cognitive overhead and unnecessarily terse.

Reply:
Not sure what this means, and its an internal function anyway. Does it mean to avoid currying?